setup-scripts/health-checker.py

```diff
@@ -114,55 +114,86 @@ async def check_health():
     print("\nChecking portal health status...")
 
     try:
-        res = requests.get("http://localhost/health-check", verify=False)
+        api = "http://localhost"
```

Comment:
When I saw this I thought you'd use the IP like in `bot_utils` but if that's not ideal then why the separate var?

Reply:
actually no reason, I reverted it